### ISSUE TITLE
Script works, pops up on other pages BUT NOT GIMKIT!

### ISSUE THREAD

**Issue by ghost** [SnowLord7/Gimkit-Hack#23]
everything worked except for gimkit itself

**Comment by Killer360556** [comment]
same

**Comment by thatrandomperson5** [comment]
Same

**Comment by zgr2575** [comment]
provide an error and proof of this for a possible fix

**Comment by yes829** [comment]
![image](https://user-images.githubusercontent.com/121134664/208772692-94d2f1fb-8ba4-4b20-bc15-f1d831166f86.png)

**Comment by blookethacksonlyyy** [comment]
yep javascript:(function()%7B(function()%20%7Blet%20x%20%3D%20document.createElement(%22script%22)%3Bx.src%20%3D%20%22https%3A%2F%2Fcdn.jsdelivr.net%2Fgh%2FSnowLord7%2FGimkit-Hack%40master%2Findex.js%22%3Bx.onload%20%3D%20function%20()%20%7B%20x.remove()%3B%20%7D%3Bdocument.body.appendChild(x)%3B%7D)()%7D)()

**Comment by belugaHackz** [comment]
Its because your console is restricted. Head over to LIGHTHOUSE DEV CONSOLE GOOGLE and change to developer mode. That should give you the permissions. Let me know if you have any more problems!
